app_integrations/apps/duo.py

```diff
@@ -116,13 +114,11 @@ def _get_duo_logs(self, hostname, full_url):
         if not headers:
             return False
 
-        # Perform the request and get the list of logs
-        response = requests.get(full_url, headers=headers, params=params)
-
-        if not self._check_http_response(response):
-            return False
+        # Make the request to the api, resulting in a bool or dict
+        response = self._make_request(full_url, headers=headers, params=params)
```

Comment:
If `response` is False, you'll get a `KeyError` on the next line

Reply:
Nice good catch. Making that fix